timetracker/entries/migrations/0003_auto_20150726_0653.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.utils.timezone
+
+
+def retain_clients(apps, schema_editor):
+    Project = apps.get_model("entries", "Project")
+    Client = apps.get_model("entries", "Client")
+    for project in Project.objects.all():
+        # Use get_or_create to ensure that clients are duplicated
+        client, created = Client.objects.get_or_create(name=project.client)
+        if created:
+            client.save()
+
+def retain_work_periods(apps, schema_editor):
+    Entry = apps.get_model("entries", "Entry")
+    WorkPeriod = apps.get_model("entries", "WorkPeriod")
+    for entry in Entry.objects.all():
+        workPeriod = WorkPeriod(start=entry.start, stop=entry.stop)
```

Comment:
The fact that at this point you're not using `get_or_create` to check if there is an existing `WorkPeriod` with the same time highlights what @funkybob is saying below.

These times are specific to a single `Entry` and won't be shared.